charts/rekor/Chart.yaml

```diff
@@ -4,8 +4,8 @@ description: Part of the sigstore project, Rekor is a timestamping server and tr
 
 type: application
 
-version: 0.3.21
-appVersion: 0.12.2
+version: 0.3.22
```

Comment:
Should we bump the version to 0.9.9 to signify a prerelease?

Reply:
Just keeping adding 9s if we have more prereleases :)

Reply:
I had a look and I believe we could create a Chart with `version: 1.0.0-rc.0`, cc @cpanato

Reply:
yes we should be able to do that

Reply:
thanks, updated!